src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -1763,6 +1763,16 @@ NEVER_INLINE Value ByteCodeInterpreter::getObjectPrecomputedCaseOperationCacheMi
         return obj->get(state, ObjectPropertyName(state, code->m_propertyName)).value(state, receiver);
     }
 
+    if (UNLIKELY(obj->isTypedArrayObject())) {
+        // disable inlineCache when property name is canonical-numeric-index only
+        // because in that case, TypedArray's own get/set internal method should be invoked.
+        double index = code->m_propertyName.canonicalNumericIndexString(state);
```

Comment:
IMO we can use `GetObjectOpcodeSlowCase` opcode for eval `*.Infinity`.
calling isTypedArrayObject() virtual function every time is too expensive